server/routerlicious/packages/routerlicious-base/src/alfred/runner.ts

```diff
@@ -82,6 +83,22 @@ export class AlfredRunner implements IRunner {
         const isTokenExpiryEnabled = this.config.get("auth:enableTokenExpiration");
         const isClientConnectivityCountingEnabled = this.config.get("usage:clientConnectivityCountingEnabled");
         const isSignalUsageCountingEnabled = this.config.get("usage:signalUsageCountingEnabled");
+        let redisCache: ICache;
+        if (this.config.get("alfred:enableConnectionCountLogging")) {
```

Comment:
Why in `Runner` and not `RunnerFactory` where we define all the other redis instances?

Reply:
Yeah, I thought about it. It seemed like the Runner param list is growing long. :p And since we do access config variables in runner as well, I just created the redis instance there.
Do you see any issues with this one? I can move it to runnerFactory, adding yet another param. :/